docs/dashboards/types-of-components.md

```diff
@@ -0,0 +1,42 @@
+Types of components
+===
+
+Components are the building blocks of the dashboard layout. There are numbers of types of components, which you can use out of the box. Some of them have come with some default configuration, and some components which leave you some flexibility so that you can configure them all by yourself.  You define which type of the component you want to use by defining its “type” property in the configuration object.
+Each component apart from the most basic one, which is HTMLComponent, needs to be imported with the `dashboard-plugin.js` module.
+
+Here is the overview of most important parameters, that can be defined for a component:
+* Id - The unique id of the component, which is later used to identify it by dashboard and/or used to set CSS styles.
+* Cell - id of the cell, in which the component should be placed
+* Class - CSS class
+* Type - the type of the component.
+* Events - object containing a pair of name of the event and callback function that should be called on a given event.
+* Sync - list of events, which should be synchronized between components.
+
+### HTML Component
+The most basic and generic component type. Lets you add everything which could be defiend as HTML, as well as add some custom events, but require the most configuration. The configuration is AST-like, where you can define the name of the tag, its attributes, and nested children elements. [Check out the basic HTML component demo here.](https://jsfiddle.net/gh/get/library/pure/highcharts/highcharts/samples/dashboard/demos/component-html)
```

Comment:
`Lets you add` - allow you add (sounds better IMO)
`defiend` - typo
`but require the most configuration` - it requires more configuration ?

Reply:
Well, in other components you are injecting some functionalities into the container, whereas in the HTML component, everything has to be implemented by the developer, so IMO, this is more flexible but also more complicated.